src/js/home/SubscribePanel.jsx

```diff
@@ -125,8 +126,15 @@ export default function SubscribePanel({
                   {arr[2] + ", "}
                   <i>{arr[1]}</i>
                 </td>
-                <td style={{ width: "30px" }}>
-                  <DeleteButton onClick={() => delSubs(region)} type="submit" value="X" />
+                <td style={{ width: "30px", display: "flex", justifyContent: "center" }}>
+                  <IconButton
```

Comment:
Would be maybe nice to define `DeleteButton` with prop of onClick and return our `IconButton` component